src/utils/index.ts

```diff
@@ -7,3 +9,51 @@ export const buildTokenLogoUri = (address: string) => {
     console.error('Failed to buildTokenLogoUri', e)
   }
 }
+
+export const splitQuery = async (
```

Comment:
can you explain the purpose of the function? adding a doc to the function would help